### ISSUE TITLE
Consider switching to Angular v. 1.3.x branch

### ISSUE THREAD

**Issue by arnbak** [mvindahl/angular-pan-zoom#32]

**Comment by mvindahl** [comment]
Makes sense. Will bump angular version in bower to newest 1.3.* and will do also revise the version number of the other dependencies.

**Comment by ClementVidal** [comment]
Is it possible to bump to Angular 1.4 ??
Do you think this could generate major troubles or it's just a matter of changing bower dependencies ?

**Comment by mvindahl** [comment]
I ran a few tests and I don't think it will be a major problem to run using angular 1.4. However, as far as I can tell, 1.4 i still in beta and the latest non-beta release is 1.3.13, so specifying 1.4.x as a dependency is a bit premature.

As far as the project is concerned I think I'll just go out on a limb and specify all dependencies as "latest", which, at the time of writing, will give you the 1.3.13 angular. I'll get around to it shortly.

**Comment by ClementVidal** [comment]
Yes you are right, Angular 1.4 is still in beta. Switching to Angular 1.3.13 sounds the best idea.
Can i help you to do that ?
Thanks !

**Comment by mvindahl** [comment]
Hi Clement, thanks for the offer but I'm already working on it. Just wanted to clean up a few things once I'm at it; committing bower_components to git is wrong so I want to stop doing that. Which means a few changes to demo.dev.html to make it fetch its external libraries from elsewhere. I think I've got it pretty much wrapped up, just need to find a twenty minute time slot to test it and do an updated bower release.

This weekend I expect ...

**Comment by ClementVidal** [comment]
Thanks a lot .
May i ask you why did you use "carret range selector ("angular": "^1.3.13") " instead of "tilde range selector" ? Just currious about that  ...  Thanks again !

**Comment by mvindahl** [comment]
You're welcome :)
And you are welcome to ask about why I used the caret range selector, but to be honest I don't have a terribly good reason. I considered everything from specifying fixed versions to specifying "latest". In the end I just made a pick of something in between.

But it's an interesting discussion and if anyone has a good reason to do something else, then I'd be happy to reconsider my choice. @askarby and @arnbak, what's your opinion on this?
